README.md

```diff
@@ -84,12 +84,14 @@ $q = new Clue\React\Mq\Queue(3, null, function ($url) use ($browser) {
 foreach ($urls as $url) {
     $q($url)->then(function (Psr\Http\Message\ResponseInterface $response) use ($url) {
         echo $url . ': ' . $response->getBody()->getSize() . ' bytes' . PHP_EOL;
-    });
+    }, function (Exception $e) {
+    echo 'Error: ' . $e->getMessage() . PHP_EOL;
+    }   );
```

Comment:
Nit: Too much spacing between brackets

Reply:
Oops, changed it right away. Thank you @SimonFrings. 🎉